apps/src/code-studio/pd/workshop_dashboard/components/survey_results/choice_responses.jsx

```diff
@@ -16,7 +16,8 @@ export default class ChoiceResponses extends React.Component {
     possibleAnswers: PropTypes.array.isRequired,
```

Comment:
only change in this file (same for text_responses) is to use new facilitators object to get facilitator names